starter-core/src/test/groovy/io/micronaut/starter/build/BuildTestVerifier.groovy

```diff
@@ -1,12 +1,17 @@
 package io.micronaut.starter.build
 
-import io.micronaut.starter.build.dependencies.Scope;
+
+import io.micronaut.starter.build.dependencies.Scope
 
 /**
  * You can get an instance via {@link io.micronaut.starter.build.BuildTestUtil}.
  */
 interface BuildTestVerifier {
 
+    boolean hasAnnotationProcessor(String groupId, String artifactId)
+
+    boolean hasAnnotationProcessor(String groupId, String artifactId, boolean isTest)
```

Comment:
Please, remove the boolean 
```suggestion
    boolean hasTestAnnotationProcessor(String groupId, String artifactId)
```

Reply:
see https://github.com/micronaut-projects/micronaut-starter/pull/2162/commits/ff4714f83e4e9c55413efc8f53d1dd92f4fe9eb2